src/altimate.ts

```diff
@@ -364,8 +368,10 @@ export class AltimateRequest {
     onProgress: (response: string) => void,
     timeout: number = 120000,
   ) {
+    this.throwIfLocalMode(endpoint);
     const url = `${AltimateRequest.ALTIMATE_URL}/${endpoint}`;
     this.dbtTerminal.debug("fetchAsStream:request", url, request);
+    this.throwIfLocalMode(endpoint);
```

Comment:
any reason to call this twice?

Reply:
fixed. it is copy paste mistake :) thanks for catching